backend/src/govsg/middlewares/govsg-transactional.middleware.ts

```diff
@@ -89,7 +89,9 @@ export const InitGovsgTransactionalMiddleware =
       }
       const isLanguageAvailable =
         govsgTemplate.multilingualSupport.findIndex(
-          (v) => v.languageCode === languageCode
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          (v) => v.language_code === languageCode
```

Comment:
I thought we store them as camelCase? https://github.com/opengovsg/postmangovsg/blob/f330bbca586491df2328f5686de3776cceedd895/backend/src/govsg/models/govsg-template.ts#L20

Reply:
Nope, it's `language_code`.

Reply:
@jia1 I think the one that needs fixing is the template data in our db, not the code

Reply:
If we change the template data, we'll need to change them all and fix the breakage to non-transactional flow. It's more work.

Reply:
I'll change the type of `GovsgTemplateLanguageMetadata`